src/main/java/org/kohsuke/github/GHBranchSync.java

```diff
@@ -0,0 +1,58 @@
+package org.kohsuke.github;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+public class GHBranchSync extends GitHubInteractiveObject {
+
+    private GHRepository owner;
+
+    private String message;
+
+    @JsonProperty("merge_type")
+    private String mergeType;
+
+    @JsonProperty("base_branch")
+    private String baseBranch;
```

Comment:
The object mapper translates camel-case to snake-case automatically. 

```suggestion
import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;

public class GHBranchSync extends GitHubInteractiveObject {

    private GHRepository owner;

    private String message;

    private String mergeType;

    private String baseBranch;
```